msteams-platform/resources/schema/manifest-schema.md

```diff
@@ -178,6 +178,12 @@ The following schema sample shows all extensibility options:
                   "value": "Value of the choice"
                 }
               ]
+               "taskInfo": {
```

Comment:
Hi, this is still the wrong place and comma is missing(so the json will fail the validation).

Please check this comment to add `taskInfo` to correct place
https://github.com/MicrosoftDocs/msteams-docs/issues/4185#issuecomment-922843921

Reply:
@Meghana-MSFT - could you please check the payload once?